pomodoro-sound-cli.sh

```diff
@@ -18,21 +18,41 @@ function notify() {
 }
 
 ### Show a countdown timer and a message and updates without
-### cleaning the whole screen
-## Params: seconds {Number} - Countdown time in seconds 
+### cleaning the whole screen. Also kills apps if needed
+## Params: seconds {Number} - Countdown time in seconds
+## Params: is_focus_time {String} - "FOCUS" or "BREAK" or "LONG_BREAK"
+## Params: app_names {String} - Comma separated list of apps to kill
 ## Params: message {String} - Timer description
+## Item: IFS=',' is (Internal Field Separator) set to comma
 ## Returns: {String} - "hh:mm:ss - $message"
 function countdown(){
   secs=$1
+  is_focus_time=$2
+  app_names=$3
+  message=$4
+
+  IFS=','
+  read -ra arr <<< "$app_names"
+
   shift
-  message=$*
+  
   while [ "$secs" -gt -1 ]
   do
+    if [[ $is_focus_time == "FOCUS" ]]; then
+      for app_name in "${arr[@]}"; do
+        pkill --signal STOP "$app_name"
+      done
+    fi
```

Comment:
<img width="518" alt="image" src="https://user-images.githubusercontent.com/86282911/206885542-8189f283-0191-4973-8ce1-4730d7de093a.png">
won't work in macos

Reply:
Is it fine if I use the pkill command for macos or do you you want me to find another way to disable apps? I thought this method will be as effective because it's running the signal stop command in a loop every second

Reply:
Umm the idea is nice, but running it every second might result in heavy battery usage or other resource consumption.
Test it out if that happens or not, then only we can decide

Reply:
Okay. I could edit the .desktop file for the apps after copying them to a local folder and also display an alert when we try to access the app. Does this sound like a better alternative

Reply:
Yeah that can work, 
OR what you can try is.. as soon as the app is opened you kill it rather than sending a sigkill every second.

Reply:
Sure. Will try both